src/sentry/runner/decorators.py

```diff
@@ -36,20 +37,31 @@ def inner(ctx, *args, **kwargs):
     return update_wrapper(inner, f)
 
 
-def log_level_option(default=None):
+def log_options(default=None):
     def decorator(f):
-        "Give ability to configure global logging level. Must be used before configuration."
+        """
+        Give ability to configure global logging level/format.
+        Must be used before configuration.
+        """
         import click
         from functools import update_wrapper
+        from sentry.logging import LoggingFormat
+        formats = [LoggingFormat.HUMAN, LoggingFormat.MACHINE]
 
         @click.pass_context
         @click.option('--loglevel', '-l', default=default,
             help='Global logging level. Use wisely.',
             envvar='SENTRY_LOG_LEVEL',
             type=CaseInsensitiveChoice(LOG_LEVELS))
-        def inner(ctx, loglevel=None, *args, **kwargs):
+        @click.option('--logformat', '-f', default=default,
```

Comment:
Oops, rolled that change into the next PR. Instead. Coming soon

On Wed, Jul 13, 2016 at 11:10 AM Matt Robenolt <notifications@github.com>
wrote:

> In src/sentry/runner/decorators.py
> <https://github.com/getsentry/sentry/pull/3685#discussion_r70678307>:
>
> >
> >          @click.pass_context
> >          @click.option('--loglevel', '-l', default=default,
> >              help='Global logging level. Use wisely.',
> >              envvar='SENTRY_LOG_LEVEL',
> >              type=CaseInsensitiveChoice(LOG_LEVELS))
> > -        def inner(ctx, loglevel=None, *args, **kwargs):
> > +        @click.option('--logformat', '-f', default=default,
>
> You didn't do this. :(
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/getsentry/sentry/pull/3685/files/3f74005207eb3c6326241dd2a3325969a9300910#r70678307>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABImRnZBR3e0lXmEubD5NLoRCQRGetZZks5qVSoDgaJpZM4JJu2I>
> .
>